Library/Homebrew/tab.rb

```diff
@@ -250,6 +250,11 @@ def to_s
     when true  then s << "Poured from bottle"
     when false then s << "Built from source"
     end
+    unless time.nil?
+      s << "Installed" if s.empty?
+      s << "on"
+      s << Time.at(time).strftime("%Y-%m-%d")
```

Comment:
I could change it to something like

```ruby
s << Time.at(time).strftime("on %Y-%m-%d at %H-%M-%S")
```

if time is also wanted. Full ISO 8601 doesn't make sense in a phrase or sentence, unfortunately.